### ISSUE TITLE
rocketchat-ui-master <title> tag

### ISSUE THREAD

**Issue by garg** [RocketChat/Rocket.Chat#2018]
The title tag on rocketchat-ui-master main.html file says: 

```html
<title>Rocket.Chat</title> 
```

Instead it should probably be a spacebars templating variable that pulls this from the RocketChat Administration settings Site Name.

The same title is hard coded in rocketchat-livechat/app/main.html, and rocketchat-livechat/public/head.html

**Comment by graywolf336** [comment]
Actually there is already a site name setting which populates the title
tag. :)

**Comment by garg** [comment]
Thanks for the response.

It didn't, for some reason, until I edited the title tag manually and then rebuilt the site. For example, if I linked to it from facebook, it was showing "Rocket.Chat" in the site preview blurb. 

Maybe I have listed the wrong package name but the Login Form page was still showing "Rocket.Chat" despite changing the Site Name setting.

**Comment by k0nsl** [comment]
`<title></title>` is actually never changed _at all_ when changing the site name setting simply because `Site_Name` is set on render.
Someone, correct me if I am wrong, or elaborate on it further with more details.

**Comment by graywolf336** [comment]
![2016-01-24_01-08-05](https://cloud.githubusercontent.com/assets/850391/12533827/b0b2bb06-c204-11e5-826f-d6455305ff19.gif)

When you update the Site Name setting, it changes whenever you save the setting. See the image.

**Comment by garg** [comment]
Sorry, my mistake. I misidentified the issue. If you don't mind, I'll post a question here regarding my issue because it is slightly related:

If I look at the facebook link debug page (https://developers.facebook.com/tools/debug/og/object?q=https%3A%2F%2Fpaint.chat)  for my installation of rocket.chat this is what I get:

~~~
Warnings That Should Be Fixed
Inferred Property	The 'og:url' property should be explicitly provided, even if a value can be inferred from other tags.
Inferred Property	The 'og:title' property should be explicitly provided, even if a value can be inferred from other tags.
Based on the raw tags, we constructed the following Open Graph properties
og:url	https://paint.chat/
og:type	website
og:title	Rocket.Chat
og:updated_time	1453223232
~~~

So basically, whenever someone posts a link to the site on fb, it shows "Rocket.Chat". So I guess I should make a ticket asking for the meta:tag og:title to be updated with the Site Name?

**Comment by graywolf336** [comment]
Yes, I would make an issue for that specifically. :+1:

**Comment by k0nsl** [comment]
Allow me to modestly reference this as well: https://github.com/RocketChat/Rocket.Chat/issues/868

**Comment by garg** [comment]
Thanks, k0nsl! Exactly that. :)
